web/styles/zulip.css

```diff
@@ -62,6 +62,12 @@ textarea {
     font-size: inherit;
 }
 
+/* Normalize this property. Chrome `user agent stylesheet` applies a 0.7
+   opacity on disabled select fields while Firefox doesn't change these. */
+select:disabled {
```

Comment:
Let's do this via the `settings-select` and `modal-select` type classes. We don't want any new CSS reset code.

Reply:
Added this change to `settings_select`, `modal_select` and `list_select` to avoid CSS action-at-a-distance.

```diff
diff --git a/web/styles/modal.css b/web/styles/modal.css
index 530a3181bf..c5b993b059 100644
--- a/web/styles/modal.css
+++ b/web/styles/modal.css
@@ -374,6 +374,9 @@
         /* The background-color of select elements inside modal is different than the others in
         settings pages, because the background of the modal is brighter than the setting page. */
         background-color: hsl(0deg 0% 90%);
+
+        /* This is reset for other browsers to use Chrome's opacity. */
+        opacity: 0.7;
     }
 }
 
diff --git a/web/styles/settings.css b/web/styles/settings.css
index e62e7bdd07..d3aaa4d208 100644
--- a/web/styles/settings.css
+++ b/web/styles/settings.css
@@ -371,6 +371,9 @@ td .button {
     &:disabled {
         cursor: not-allowed;
         background-color: hsl(0deg 0% 93%);
+
+        /* This is reset for other browsers to use Chrome's opacity. */
+        opacity: 0.7;
     }
 }
 ```